portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,46 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Ibiyemi's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <header>
+    <h1>
+      Ibiyemi Abiodun
+    </h1>
+  </header>
+  <section id="content">
+    <h2>
+      Hi, I'm Ibiyemi Abiodun!
+    </h2>
+    <p>
+      I'm a rising sophomore at Cornell, double majoring in electrical &
+      computer engineering, and computer science. I love cooking & baking,
+      karate, VFX, cinematography, and of course, coding.
```

Comment:
You should lean on the paragraph element containing this text to dictate when a line gets broken off instead of manually breaking it off into multiple lines. So in this case, your sentences should fill the whole line up to the 125 character limit and then you can break off to a new line. Since this PR is just about adding body text, you can add any styling in a separate PR (don't worry about doing that here).